quickstart/101-managed-instance/variables.tf

```diff
@@ -0,0 +1,64 @@
+variable "azurerm_resource_group_name" {
+  type        = string
+  description = "Enter the resource group name"
+  default     = "terraform-database-resource-group"
+}
+variable "azurerm_network_security_group_name" {
+  type        = string
+  description = "Enter the security group name"
+  default     = "mi-security-group-terraform"
+}
+variable "azurerm_virtual_network_name" {
+  type        = string
+  description = "Enter the virtual network name"
+  default     = "vnet-mi-terraform"
+}
+variable "azurerm_subnet_name" {
+  type        = string
+  description = "Enter subnet name"
+  default     = "subnet-mi-terraform"
+}
+variable "location" {
+  type        = string
+  description = "Enter the location where you want to deploy the resources"
+  default     = "eastus2euap"
+}
+
+variable "administrator_login" {
+  type        = string
+  description = "Enter Administrator name for the database"
+  default     = "VeryStrongAdministrator"
+}
+
+variable "administrator_login_password" {
```

Comment:
Since this variable has been used as password, could we add [`sensitive = true`](https://www.terraform.io/language/values/variables#suppressing-values-in-cli-output) for this variable?

Reply:
done